documentation/book/getting-started.adoc

```diff
@@ -9,12 +9,10 @@ The Docker images are available on {DockerRepository}.
 
 === Prerequisites
 
-An {ProductPlatformName} cluster is required to deploy {ProductLongName}. {ProductName} supports all kinds of clusters - from public and
-private clouds down to local deployments intended for development purposes. This guide expects that an {ProductPlatformName}
-cluster is available and the
-ifdef::Kubernetes[`kubectl` or]
-`oc` command line tools are installed and configured to connect
-to the running cluster.
+{ProductLongName} runs on {ProductPlatformName} cluster.
```

Comment:
Same here, although, we could also remove the article and the following "cluster" from [ProductPlatformName]. For example, [ProductLongName] runs on [ProductPlatformName]. That way, we wouldn't need to worry about the article